doc/source/amazon/step-zero-aws.rst

```diff
@@ -260,6 +264,15 @@ Then perform the following steps:
     
     If you really want to insure encryption is working, you can listen on port `6783` of any node. If the traffic looks like gibberish, you know it is on.
 
-    
+==============
+Shared Storage
+==============
+A shared volume is supposed to be mounted to multiple user pods, so we cannot use EBS. As an alternative, there's AWS EFS:
+
+#. `Set up a EFS storage on AWS <https://zero-to-jupyterhub.readthedocs.io/en/latest/amazon/efs_storage.html>`_
```

Comment:
we can use rst cross-references so that these links are portable for any build of the docs:

```suggestion
#. :ref:`amazon-efs`
```